### PR TITLE
table

### PR DESCRIPTION
**Pull request by alisa-bielenko**

### CODE REVIEW

File: lesson_5/index5.html [hunk 1]
```diff
@@ -23,10 +24,14 @@ <h1>Lesson_4</h1>
                 </td>
             </tr>
             <tr>
-                <td><b>Bill to</b><p>Street adress</p>
-                    <p>City, state, zip</p></td>
-                <td><b>Ship to</b><p>Street adress</p>
-                    <p>City, state, zip</p></td>
+                <td><b>Bill to</b>
```
Comment (RomanIlkiv) [inline lesson_5/index5.html:27]: it should be from a new line, like this:

```
<td>
  <b>Bill to</b>
  <p>Street adress</p>
  City, state, zip
</td>
```

File: lesson_5/index5.html [hunk 2]
```diff
@@ -23,10 +24,14 @@ <h1>Lesson_4</h1>
...
+                    <p>Street adress</p>
+                    City, state, zip
+                </td>
+                <td><b>Ship to</b>
```
Comment (RomanIlkiv) [inline lesson_5/index5.html:31]: the same here

File: lesson_5/index5.html [hunk 3]
```diff
@@ -69,5 +74,47 @@ <h1>Lesson_4</h1>
                 <td>0</td>
             </tr>
         </table>
+        <br>
+        <br>
+        <br>
+        <caption>Table 2</caption>
```
Comment (RomanIlkiv) [inline lesson_5/index5.html:80]: **caption** tag must be inside of **table** tag

File: lesson_5/index5.html [hunk 4]
```diff
@@ -69,5 +74,47 @@ <h1>Lesson_4</h1>
...
+    <table border="=1">
```
Comment (RomanIlkiv) [inline lesson_5/index5.html:81]: fix space formatting

File: lesson_5/index5.html [hunk 5]
```diff
@@ -69,5 +74,47 @@ <h1>Lesson_4</h1>
...
+            <td>Electrical</td>
+            <td>34567</td>
+        </tr>
+        <tr>
+            <td>4</td>
+            <td>Rick</td>
+            <td>Civil</td>
+            <td>45678</td>
+        </tr>
+    </table>
```
Comment (RomanIlkiv) [inline lesson_5/index5.html:118]: table tag is inside of body, so should be nested by spaces as all tags

File: lesson_5/index5.html (hunk 1 above)
Comment (alisa-bielenko) [reply lesson_5/index5.html:27]: ok

File: lesson_5/index5.html (hunk 2 above)
Comment (alisa-bielenko) [reply lesson_5/index5.html:31]: ok

File: lesson_5/index5.html (hunk 3 above)
Comment (alisa-bielenko) [reply lesson_5/index5.html:80]: ok

File: lesson_5/index5.html (hunk 4 above)
Comment (alisa-bielenko) [reply lesson_5/index5.html:81]: ok

File: lesson_5/index5.html (hunk 5 above)
Comment (alisa-bielenko) [reply lesson_5/index5.html:118]: ok

**Pull request closed without merging**